tools/actions/composites/bot/action.yml

```diff
@@ -63,7 +72,10 @@ runs:
         VERBOSE_FILE: botreport/logs.txt
         GITHUB_RUN_ID: ${{ github.run_id }}
         GITHUB_WORKFLOW: ${{ github.workflow }}
-        BOT_FILTER_FAMILY: ${{ inputs.BOT_FILTER_FAMILY }}
+        BOT_FILTER_FAMILIES: ${{ inputs.BOT_FILTER_FAMILIES }}
```

Comment:
I see you renamed the BOT_FILTER_FAMILY with BOT_FILTER_FAMILIES.
You can check all the BOT_FILTER_FAMILY in our codebase. I found maybe you still forgot to update some yml files.

Reply:
Could be the case ! Will check

Reply:
Indeeed, nice catch, done